vpp-manager/startup/startup.go

```diff
@@ -106,7 +108,7 @@ const (
 )
 
 func GetVppManagerParams() (params *config.VppManagerParams) {
-	params = &config.VppManagerParams{}
+	params = &config.VppManagerParams{InterfacesSpecs: []config.InterfaceSpec{{IsMain: true /* first interface is main */}}}
```

Comment:
Since you're overwriting params.InterfaceSpecs at the end, maybe instead of initializing params.InterfacesSpec[0] here, it would be a bit cleaner & more explicit to just define a variable `mainInterfaceSpec` of type `config.InterfaceSpec` here that you would use through the function when parsing the different env vars.